src/milo.h

```diff
@@ -128,10 +128,6 @@ struct DiyFp {
         unsigned long index;
         _BitScanReverse64(&index, f);
         return DiyFp(f << (63 - index), e - (63 - index));
-#elif defined(__GNUC__) ||                                                     \
-    (__clang__ && defined __has_builtin && __has_builtin(__builtin_clzll))
-        int s = __builtin_clzll(f);
-        return DiyFp(f << s, e - s);
 #else

```

Comment:
Is this an optimization?

Reply:
No idea, this is not my code, but it certainly looks so :)